src/components/app-configure/chain-management/AppChainManagementList.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { ref, toRefs, computed } from 'vue'
+import { ref, toRefs, computed, onMounted } from 'vue'
```

Comment:
Where is this onMounted used?

Reply:
removed. [ref](https://github.com/arcana-network/developer-dashboard/pull/394/commits/ba52194c67684c5782f82ae55e98612762e0242f)